tests/test_todo.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+import os
+from dotenv import load_dotenv, find_dotenv
+from todo_app import app
+from unittest.mock import Mock, patch
+
+@pytest.fixture
+def client():
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+    test_app = app.create_app()
+    with test_app.test_client() as client:
+        yield client
+
+class Test_Index_Page:
+
+    @patch('requests.get')
+    def test_index_page(mock_get_requests, client):
+        mock_get_requests.side_effect = mock_request
+        response = client.get('/')
```

Comment:
You should add some assertions here, e.g. check for a 200 response code and that your test data is included in the response.